### PR TITLE
Completes FakeOS Windows 98 Project

### PR DESCRIPTION
**Pull request by PegliOne**

### CODE REVIEW

File: js/app-functions/greetings.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  if (randomNum < 0.25) {
+    greeting = "Hello There!";
+  } else if (randomNum < 0.5) {
+    greeting = "Bonjour!";
+  } else if (randomNum < 0.75) {
+    greeting = "Hola!";
+  } else {
+    greeting = "Hey, what's up?";
+  }
+  return greeting;
```
Comment (ABaulderstone) [inline js/app-functions/greetings.js:20]: An alternative approach might be something like 
```js
 const greetings = ["Hey", "Hola", "Hi"];
 const randomIndex = Math.floor(Math.random() * greetings.length); 
 return greetings[randomIndex]
```

File: js/app-functions/number-guess.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const checkGuess = (correctNumber) => {
+  const numberInput = document.querySelector("#numberInput");
+  let guess = Number(numberInput.value);
+
+  if (guess === correctNumber) {
+    guessResult.textContent = "Correct!";
+  } else {
+    guessResult.textContent = "Keep guessing!";
+  }
```
Comment (ABaulderstone) [inline js/app-functions/number-guess.js:19]: Could do a higher/lower system here, and maybe keep track of the guesses?

File: js/menubar.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+};
+
+export const getTime = () => {
+  const time = new Date();
+  const hours = time.getHours();
+  const minutes =
+    time.getMinutes() < 10 ? `0${time.getMinutes()}` : time.getMinutes();
+  const amOrPM = hours < 12 ? "AM" : "PM";
+  return `${hours}:${minutes} ${amOrPM}`;
+};
```
Comment (ABaulderstone) [inline js/menubar.js:21]: nice. You can also use toLocaleTimeString()

File: js/modals.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export const getModal = (event) => {
+  const targetId = event.currentTarget.getAttribute("value");
+  const targetModal = document.querySelector(`#${targetId}`);
+  return { targetId, targetModal };
+};
```
Comment (ABaulderstone) [inline js/modals.js:5]: this is cool

**Review by ABaulderstone — APPROVED** [review]
Great job, very impressed with the scss and some very well organised javascript here. 
A few suggestions to make the apps a bit more advanced but the core of the project is very much done

File: js/app-functions/number-guess.js (hunk 2 above)
Comment (ABaulderstone) [reply js/app-functions/number-guess.js:19]: You can add styles as well, like green for correct, orange if it's close and red if it's far off

File: js/app-functions/greetings.js (hunk 1 above)
Comment (PegliOne) [reply js/app-functions/greetings.js:20]: Thanks Alex, I was able to implement this successfully. The only change I had to make was replacing "const randomIndex" with "let randomIndex", so the greeting wouldn't be the same each time.

File: js/menubar.js (hunk 3 above)
Comment (PegliOne) [reply js/menubar.js:21]: I have updated it to use toLocaleTimeString() for the time itself, but I had to add the amOrPm after it.

**Pull request merged**